### ISSUE TITLE
Two instances of a very deep cut that resulted in data loss

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/BakingTray#212]
This has never happened before. There were two cases of BS apparently cutting a very thick (300 to 500 micron) section, which sank down to the bottom of the enclosure and meant that we lost data from that region. This happened in `ERAB24_2b` and `ERAE33_1g`. The slice was deeper at the back than at the front. Apparently it looked like a 5 to degree wedge. 

Working hypothesis is that cut speed was not set and the blade drove into the sample at max speed. Try to verify. Likely we can tell from the logs. If so, the fix will be to put into place a thorough check of cut speed before cutting.

**Comment by raacampbell** [comment]
For `ERAE33_1g`, it happened from section_138_01.tif (or 1371). Note that images were a bit wobbly in all sections afterwards even after the thick slice.

**Comment by raacampbell** [comment]
For `ERAB24_2b`, it happened twice. Once from section_51_01.tif (or 501 sequentially), note there was a shift in brain position compared to section 50_10.tif, but later came back as normal again. It happened again from `section_95_01.tif` (or 941)

**Comment by raacampbell** [comment]
None of the slice times look peculiar and nothing else I can so far see is odd in the logs. However, today the Z stage was commanded to move from 0 to 22 mm and the stepper stalled. This caused it to end up several mm short but still report it's at 22 mm. Of course this does not explain the problem seen above, but it does raise the real possibility that the stage is doing odd stuff that isn't being recorded since there is no encoder.

**Comment by raacampbell** [comment]
Not happened again.

**Comment by raacampbell** [comment]
I think it happened again today on sample 972991 from SL. I'm at work and able to look. I think it's most consistent with the speed being wrong and cutting too quickly.

**Comment by raacampbell** [comment]
Tried to replicate as follows:
* Take a series of 100 micron cuts. at 0,5 mm/s
* Take a 1.1 mm cut a 0.5 mm/s

The resulting slice surface is very nice and at the same level as the blade. So this looks nothing like what I saw 8 days ago. 
Ah! But if the slide is loose in the holder then it can pull up. I'm pretty sure this is what happened these other times also. So We need a solution that physically holds the slide better.
